python/guide/getting-started/to-mxnet/pytorch.md

```diff
@@ -54,19 +50,17 @@ y = x + 1
 y
 ```
 
-The main difference apart from the package name is that the MXNet's shape input
-parameter needs to be passed as a tuple enclosed in parentheses as in NumPy.
+The main difference apart from the package name is that the MXNet's shape input parameter needs to be passed as a tuple enclosed in parentheses as in NumPy.
+
+Both frameworks support multiple functions to create and manipulate tensors / NDArrays. You can find more of them in the documentation.
 
 ## Model training
 
-Let's look at a slightly more complicated example below. Here we create a
-Multilayer Perceptron (MLP) to train a model on the MINST data set. We divide
-the experiment into 4 sections.
+After covering the basics of data creation and manipulation, let's dive deep and compare how model training is done in both frameworks. In order to do so, we are going to solve image classification task on MNIST data set using Multilayer Perceptron (MLP) in both frameworks. We divide the task in 4 steps.
 
 ### 1 --- Read data
```

Comment:
Would avoid this style of section title. We don't use it anywhere else. Stick with "### 1. Read Data". Same applied for other section headings.